src/app/api/admin/[...slug]/inputFilters.js

```diff
@@ -0,0 +1,7 @@
+import { MODEL_SEARCH_FIELDS } from './consts';
```

Comment:
🟠 why don't put `consts` and `inputFilters` files content in route is it used only there? having 2 files for 4 + 7 lines of code looks a bit like overkill

Reply:
Ok, i'll change it

Reply:
@alex-popov-tech isn't it better to keep only route handler in route.js file? Also there may be more consts or service functions related only to this route in future. I suggest to rename inputFilters.js to utils.js or smth like this

Reply:
whats important in `route.js` is name - as next will look for it, and few exports of specific format

all other code is alright to keep there

and since there are no suggestions or best practices for routes files, i would suggest to treat it like plain js file, where you have 'main' part, and 'support' functions/variables which are needed to make 'main' more readable and self-explanatory

creating more files with name which can be treated differently ( `inputFilters` makes sence in context of that plugin settings, which is inside the route, but it does not make sence to `route.js` ) brings more complications than few extra lines into route file itself

but if you strongly believe that should be separated - i will accept that too :)

Reply:
Ok, you got the point. I'll change it